src/redux/actions/projects/createProject.js

```diff
@@ -24,8 +25,12 @@ const createProject = (
 
   const newProjectUuid = uuidv4();
 
-  // Always create an experiment for a new project
-  // required because samples DynamoDB require experimentId
+  if (api.CURRENT_VERSION === api.possibleVersions.V2) {
+    await dispatch(createExperiment(newProjectUuid, newExperimentName));
```

Comment:
Do we need to await here?

Reply:
Why not? It is better to link the async functions to its internal async processes when possible so that the async handling can be brought up to the containing function, right?